### ISSUE TITLE
Bug: site.related_posts not work (0.15.beta)

### ISSUE THREAD

**Issue by zealot128** [bridgetownrb/bridgetown#78]
Followed the documentation to show related posts below, but the variable ``{{ site.related_posts }}`` (as well as ``{{ page.related_posts }}``, ``{{ related_posts }}`` is always empty (Output of all ``{{ site.related_posts | inspect }}`` is nil).

**Bridgetown Version**:

bridgetown 0.15.0.beta3 "Overlook"

**To Reproduce**

- having couple of posts

```html
<!-- _layouts/post.html -->
---
layout: default
---
   <strong>Related posts</strong>
    <ul>
      {% for post in site.related_posts %}
        <li>
          <a href='{{post.url}}'>{{post.title}}</a>
          <small>{{post.date}} &ndash; {{post.tags}}</small>
        </li>
      {% endfor %}
    </ul>
```

**Fix/Workaround**

Add ``, :related_posts`` to https://github.com/bridgetownrb/bridgetown/blob/5aca30ff8cf84334c1e23f9e624ada76a33f17a5/bridgetown-core/lib/bridgetown-core/drops/document_drop.rb#L15

Then, the variable ``{{ page.related_posts }}`` works (But not the site.related_posts according to the documentation).

**Comment by jaredcwhite** [comment]
Thanks for the report @zealot128. :+1: We'll look into this.

**Comment by jaredcwhite** [comment]
Your fix/workaround is exactly what was needed! Also, the documentation was incorrect…it _is_ supposed to be `page.related_posts`, not `site.related_posts`. Both are now included in 0.15.0.beta4 release.

**Comment by zealot128** [comment]
Awesome, keep up the good work! :)
